Extension/package.json

```diff
@@ -1882,6 +1882,32 @@
         "./debugAdapters/mono.linux-x86_64"
       ]
     },
+    {
+      "description": "Mono Runtime (Linux / armhf)",
+      "url": "https://aka.ms/mono-runtime-linux-arm",
```

Comment:
Should these be fwlinks instead of aka.ms links?  I don't have a strong preference, but I think it's easier to manage links when they're all on the same dashboard.

Reply:
I was thinking it would be easier to be more descriptive with aka.ms links since then you know what it points to but i also didn't want to update the rest of them today.